src/server/server_family.cc

```diff
@@ -861,6 +861,7 @@ void ServerFamily::Init(util::AcceptServer* acceptor, std::vector<facade::Listen
   config_registry.RegisterMutable("tls_ca_cert_dir");
   config_registry.RegisterMutable("replica_priority");
   config_registry.RegisterMutable("lua_undeclared_keys_shas");
+  config_registry.RegisterMutable("list_rdb_encode_v2");
```

Comment:
no need, imho

Reply:
if we have any prod issues with the new impl we will be able to recover